test/integration/scenarios/04.workflow/magefile.go

```diff
@@ -243,3 +251,40 @@ func writeYamlStringsToFile(yamlString string, filePath string) error {
 
 	return nil
 }
+
+func modifyYAML(v string) error {
+	// Read the YAML file
+	data, err := os.ReadFile("../../../localenv/symphony-ghcr-values.yaml")
+	if err != nil {
+		return err
+	}
+
+	// Unmarshal the YAML data into a map
+	var values map[string]interface{}
+	err = yaml.Unmarshal(data, &values)
+	if err != nil {
+		return err
+	}
+
+	// Modify the 'api' fields
+	if api, ok := values["api"].(map[interface{}]interface{}); ok {
+		api["labelKey"] = v
+		api["labelValue"] = v
+	} else {
+		return fmt.Errorf("'api' field is not a map")
+	}
+
+	// Marshal the map back into YAML
+	data, err = yaml.Marshal(values)
+	if err != nil {
+		return err
+	}
+
+	// Write the modified YAML data back to the file
+	err = os.WriteFile("../../../localenv/symphony-ghcr-values.yaml", data, 0644)
```

Comment:
This will change the default value.yaml, then the following tests won't test with default setting. 
It is better you can use different value file, you can change the function in https://github.com/eclipse-symphony/symphony/blob/dccc41a2c954a7f403b7c4c173fb1498f3708c5d/test/localenv/magefile.go#L130-L158 to respect the value file from environment var.